docs/contributors/code/react-native/internationalization-guide.md

```diff
@@ -0,0 +1,75 @@
+# React Native Internationalization Guide
+
+The native version of the editor references two types of string:
+1. Strings used in web and native platforms.
+2. Strings used only in the native platform.
+
+Regarding the first type, these strings are translated following the same process described for the web version in [this guide](https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/internationalization.md), however for the latter, it's required to provide your own translations.
+
+## Extract strings only used in the native platform
+
+In order to identify these strings, you can use the script [`extract-used-strings`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/bin/extract-used-strings.js) located in `packages/react-native-editor/bin/extract-used-strings.js` to generate a JSON object that contains all the strings referenced including the platforms where they are used, as well as the files that reference it. Here you can see the format:
+```
+ {
+	"gutenberg": {
+	  "<string>": {
+	    "string": String value.
+		"stringPlural": String value with its plural form. [optional]
+		"comments": Comments for translators. [default value is an empty string]
+		"reference": Array containing the paths of the source files that reference the string.
+		"platforms": Array containing the platforms where the string is being used, values are "android" | "ios" | "web".
+	  },
+	  ...
+	},
+	"other-domain-plugin": {
+	  ...
+	},
+    ...
+}
+```
+
+This command also supports passing extra plugins, in case the React Native bundle of the editor is generated including other plugins.
+
+Filtering from this data the strings that doesn't include the "web" platform, it will provide the list of strings that are required to be translated. Once the translations for these strings are ready, they can be injected into the editor via the `translations` initial prop when it’s initialized:
+- [Android reference](https://github.com/WordPress/gutenberg/blob/72854b4d6b09bd7fb7f996a5c55dd3cc0613ddf8/packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt#L34)
+- [iOS reference](https://github.com/WordPress/gutenberg/blob/72854b4d6b09bd7fb7f996a5c55dd3cc0613ddf8/packages/react-native-bridge/ios/GutenbergBridgeDataSource.swift#L39-L43)
+
+The translations passed through this initial prop will be merged with the ones already included in the editor.
+
+### NPM command
+
+Extract used strings:
+```sh
+npm run native i18n:extract-used-strings -- "$PWD/used-strings.json"
+```
+
+***NOTE:** We need to pass absolute paths, otherwise it uses `packages/react-native-editor` as root path  for relative paths.*
```

Comment:
minor: extra whitespace between between "path" and "for" near the end of the sentence

Reply:
I've applied your suggestion in https://github.com/WordPress/gutenberg/pull/37918/commits/a2d2e65b20fcf40c090ef92c99fca761e08857e0.